### DISCUSSION TITLE
I can't pass cloudflare DDoS protection on qutebrowser.

### DISCUSSION THREAD

**Discussion by amano-kenji** [qutebrowser/qutebrowser#7318] [Q&A] [answered]
These websites use cloudflare DDoS protection which should not be used.

* https://tradeogre.com
* https://gitlab.com log in

What can anyone do about this? cloudflare also blocks GNU Icecat browser.

**Comment by The-Compiler** [comment, marked as answer]
It's unlikely that anyone other can Cloudflare can do something about it.

**Reply by The-Compiler** [reply]
Also see #7208
